src/domain/page/pageHeader/pageHeader.module.scss

```diff
@@ -0,0 +1,15 @@
+@import 'colours';
+@import 'spacings';
+
+.mainLink {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
This style is not needed because the element (`a`) width is based on the content (there is no free space to distribute), so we are fine with the default value `flex-start`, so no additional style needed.